### ISSUE TITLE
WCAG 2.1: High Contrast mode - Buttons' inner background color has insufficient contrast

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-web#763]
## Describe the bug

Same as #762 , in high contrast mode

## Screenshots
![image](https://user-images.githubusercontent.com/22068434/59233631-19798a00-8b9e-11e9-8036-4df3d30e91d9.png)

![image](https://user-images.githubusercontent.com/22068434/59233659-38781c00-8b9e-11e9-9053-2bfaadc88cc9.png)

## Extension

- Accessibility Insights for Web - prod
- Browser Version: Chrome 75.0.3770.80
- Target Page: Any page

**Comment by cheade** [comment]
### **High contrast mode secondary buttons**
In my original Figma designs, I had our secondary buttons styled as such:

![image](https://user-images.githubusercontent.com/40274682/59383943-e3f7ac80-8d15-11e9-9e6e-791158c1908b.png)

(White background with black text.)

However, In accordance with WCAG 2.1 and to match my suggestion for #762, I suggest:

![image](https://user-images.githubusercontent.com/40274682/59384035-186b6880-8d16-11e9-92f8-eca341a833d9.png)

**Button styling**
background-color: #000000;
border-color: #FFFFFF;
border-radius: 2px;

**Text**
color: $primary-text-inverse

**Comment by ferBonnin** [comment]
same as #762, what would we show for focus?

**Comment by cheade** [comment]
Would something like this work? 

![image](https://user-images.githubusercontent.com/40274682/59394828-0f8a8f00-8d36-11e9-918d-a514ce0b2aa7.png)

I'm not sure if we can include the nuanced states of hover and pressed in high contrast mode.

**Comment by ferBonnin** [comment]
per [1.4.11](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) the states of the button ( on hover, focus, select, press, etc) also need to have sufficient color contrast

**Comment by cheade** [comment]
@ferBonnin great, then these all should work! Each of the states is above 7:1 with their adjacent colors. The rule doesn't apply to, for example, hover contrast against the rest contrast since they aren't going to appear next to each other.

**Comment by ferBonnin** [comment]
perfect!
Couple of questions, what colors are those in the palette?
Would we use the primary or the secondary/standard?

**Comment by cheade** [comment]
We would use the secondary button in this specific scenario, but I wanted to design a primary button as well. (I'm not sure if we have any primary buttons other than the 'accept telemetry' pop-up, but I wanted to be prepared)

Here is the [Figma fil](https://www.figma.com/file/Wj4Ggf6GGQBQkiDIaHfXRX2B/Accessibility-Insights-Style-guide?node-id=13%3A153)e where all of the appropriate colors can be pulled from.

**Comment by ferBonnin** [comment]
@cheade whats the difference between rest and on hover?

**Comment by cheade** [comment]
@ferBonnin rest and hover vary slightly by color. It's more noticeable on hover when the button changes below a cursor. Rest background is #000000 and the hover background is #272727

**Comment by ferBonnin** [comment]
@cheade per #762 do we want dotted line here too?

**Comment by cheade** [comment]
For focus states, yep!

The [Figma styles](https://www.figma.com/file/Wj4Ggf6GGQBQkiDIaHfXRX2B/Accessibility-Insights-Styles?node-id=13%3A153) file is updated

**Comment by ferBonnin** [comment]
@iamrafan looks like this has been addressed in OF V7, can you confirm?

**Comment by iamrafan** [comment]
@ferBonnin Yes, this issue has been fixed
